docs/buildah-build.1.md

```diff
@@ -645,9 +645,17 @@ Valid _mode_ values are:
 
 Do not use existing cached images for the container build. Build from the start with a new set of cached layers.
 
+**--no-hostname**
+
+Do not create _/etc/hostname_ for RUN instructions.
+
+By default, Buildah manages _/etc/hostname_, adding the container's own hostname.
```

Comment:
I'm on the fence about adding the word `file` after /etc/hostname here and below.